src/ui/MissingTerm/MissingTerms.ts

```diff
@@ -217,5 +215,11 @@ export class MissingTerms extends Component {
       allMissingTerms[index].removeAttribute('style');
     }
   }
+
+  private logUAMissingTerm(term: string) {
```

Comment:
Should this method be in `MissingTermManager.ts`? Seems inconsistent with the remove event.
Also, same comment on mentioning the type of event: `logMissingTermAddedEvent`

Reply:
The manager is created with the `searchinterface` all it does is create bindings when building a query and manage the breadcrumb. So it can't log a UA when a missing term is added.

Instead,  the event is log when the missing term element is clicked in the 

If we really want, I guess I could do a static method in the manager and call it here?